app/Models/DocumentRequest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class DocumentRequest extends Model
+{
+    use HasFactory;
+
+    public $timestamps = true;
+
+    protected $fillable = [
+        'document_type',
```

Comment:
There is currently no class for documents, so I am not sure how to replace this string field with Role-like tags. We could maybe move the functions from the documentController to a dedicated class and link it to the document to the request.

Reply:
Yes, we might need further planning for this.
I think this should be easier than roles. Since we do not necessarily need a many to many relation between users and documents, we don't need a table or a class for the documents. (What do you think, @luksan47?).

But we need a constant lists of the possible document types (see checkout or role models) in the DocumentRequest class.
Eg. when creating a document request: `'document_type' => DocumentRequest::STATUS_CERTIFICATE` is nicer, and you can set `...::ALL` in the factory.

My other problem is that the basic document types can be generated by knowing the type, the user id and the time, but we may need additional information (comments, justification, ... for a conference participation or even for custom documents). @luksan47 Is adding a simple `note` field for the table enough? If so, please do that, and we can work with it later.

Reply:
Agree with the document type being a constant, we use this at a few other places as well. No need to create another file or class, it's good here.

Regarding the other problem, let's not add any unnecessary fields we would not sue right now. If we'd need a new column, we could always add it later.